src/main/java/be/seeseemelk/mockbukkit/inventory/InventoryMock.java

```diff
@@ -526,8 +527,10 @@ public void clear()
 	@Override
 	public int close()
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+		int count = this.viewers.size();
+		System.out.println(viewers);
+		Lists.newArrayList(this.viewers).forEach(HumanEntity::closeInventory);
```

Comment:
Why is the list cloned?

Reply:
This is the way Paper does it, so I kept it